melosys-eessi-app/src/main/java/no/nav/melosys/eessi/kafka/consumers/SedMottattConsumer.java

```diff
@@ -56,6 +64,7 @@ public void sedMottatt(ConsumerRecord<String, SedHendelse> consumerRecord) {
             sedMetrikker.sedMottattFeilet(sedHendelse.getSedType());
             String message = e.getMessage() != null ? e.getMessage() : e.getClass().getSimpleName();
             log.error("sedMottatt feilet: {}\n{}", message, consumerRecord, e);
+            throw e;
```

Comment:
Hvordan vil dette påvirke muligheten for re-kjøre fra logger? 
Får vi samme feilen logget 5 ganger

Reply:
Ja, vi kommer til å få loggen 5 ganger hvis det feiler 5 ganger.. Blir det litt rotete i loggen kanskje